Source/DTDLv2/RealEstateCore/Asset/Equipment/ICTEquipment/ICTHardware/ICTHardware.json

```diff
@@ -0,0 +1,12 @@
+{
+  "@id": "dtmi:org:w3id:rec:ICTHardware;1",
```

Comment:
If we've removed the "Server" subclass, what other thing would be "ICT Hardware" but not "ICT Equipment"? Might it be worthwhile to keep Server and drop this class?

Reply:
I think I forgot adding the Server. Will fix.

The I suppose there can be equipments such a power supplies / UPS that someone would want to add!?

Reply:
Fixed with new commit.